solidserver/data_source_ip_pool.go

```diff
@@ -0,0 +1,128 @@
+package solidserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"net/url"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceippool() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceippoolRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The name of the pool.",
+				Required:    true,
+			},
+			"subnet": {
+				Type:        schema.TypeString,
+				Description: "The subnet associated to the pool.",
+				Required:    true,
+			},
+			"space": {
+				Type:        schema.TypeString,
+				Description: "The space associated to the pool.",
+				Required:    true,
+			},
+			"start": {
+				Type:        schema.TypeString,
+				Description: "The fisrt address of the pool.",
+				Computed:    true,
+			},
+			"end": {
+				Type:        schema.TypeString,
+				Description: "The last address of the pool.",
+				Computed:    true,
+			},
+			"size": {
+				Type:        schema.TypeString,
+				Description: "The size of the pool.",
+				Computed:    true,
+			},
+			"cidr": {
```

Comment:
By CIDR, do you mean the prefix-size ? If so, I would suggest to name this field "prefix-size" for consistency.

Reply:
I mean prefix length...